src/freertos_drivers/arduino/Can.cxx

```diff
@@ -36,5 +36,9 @@
 #include "can_frame.h"
 #include <cstdint>
 
+namespace openmrn_arduino {
```

Comment:
I believe the trailing `{` should be on the next line, this likely should be applied elsewhere as well